server/data/activitiesApiClient.test.ts

```diff
@@ -681,13 +684,13 @@ describe('activitiesApiClient', () => {
         inCell: false,
         startDate: '2023-05-16',
         appointments: [
-          { prisonerNumber: 'A1349DZ', startTime: '13:30', endTime: '14:30' },
-          { prisonerNumber: 'A1350DZ', startTime: '15:00', endTime: '15:00' },
+          { prisonerNumber: 'A1349DZ', startTime: '13:30', endTime: '14:30', comment: '' } as IndividualAppointment,
```

Comment:
I noticed this recently and it's my mistake, but I think the API schema is wrong here and comment should probably be optional?

Either way this change is fine though.